frontend/public/index.html

```diff
@@ -4,6 +4,10 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" href="%PUBLIC_URL%/Logo.png" />
+    <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
```

Comment:
https://mui.com/material-ui/icons/#font-material-icons
to be able use icons as strings in Icon component